geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/filter/FilterFunctions.scala

```diff
@@ -0,0 +1,41 @@
+/***********************************************************************
+ * Copyright (c) 2013-2020 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.utils.geotools.filter
+
+import org.opengis.parameter.Parameter
+
+import scala.reflect.ClassTag
+
+object FilterFunctions {
```

Comment:
This helper amuses me.  Did it save something up above?

Reply:
well FilterFunctionImpl has a `parameter` helper, but it doesn't expose `optional` so i made a new one :)